### PR TITLE
:sparkles: Add BackButton Component, externalized assets (images and icons), modified globals CSS

### LINKED ISSUE #13: Build components: page - Meta: BackButton
Bildschirmfoto 2021-06-09 um 18 00 00

Build back button and integrate history function.

### PR DESCRIPTION
**Pull request by daebert**
fixes #13
http://localhost:6007/?path=/story/component-backbutton--back

### CODE REVIEW

File: src/app/assets/Icons/BackArrowIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      xmlns="http://www.w3.org/2000/svg"
+      width="26.394"
+      height="26.3"
+      viewBox="0 0 26.394 26.3"
+    >
+      <path
+        d="M14.942,26.906l-1.288,1.322a1.362,1.362,0,0,1-1.967,0L.406,16.659a1.449,1.449,0,0,1,0-2.018L11.686,3.067a1.362,1.362,0,0,1,1.967,0l1.288,1.322a1.457,1.457,0,0,1-.023,2.042L7.926,13.266H24.6A1.408,1.408,0,0,1,26,14.695V16.6A1.408,1.408,0,0,1,24.6,18.029H7.926l6.993,6.835A1.446,1.446,0,0,1,14.942,26.906Z"
+        transform="translate(0.247 -2.497)"
+        fill="#707070"
+        stroke="#707070"
```
Comment (Dev-dfm) [inline src/app/assets/Icons/BackArrowIcon.tsx:15]:
```suggestion
Maybay use your colors defined from :root in globals.css
```

**Comment by lmachens** [comment]
@daebert the deployment is failing. The link localhost:6007 you shared is for local development only (and only works, if you clone and run your app).

**Review by lmachens — COMMENTED** [review]
You have some assets which are never used.  
The naming convention for these pictures are different (Capital or pascalCase?)
`dancing.png` vs `Football.png`.
Any reason why the `Images` folder has an upper case `I`?

File: src/app/assets/Icons/BackArrowIcon.tsx (hunk 1 above)
Comment (daebert) [reply src/app/assets/Icons/BackArrowIcon.tsx:15]: Done, thanks!

**Review by Niklas-Schaefers — APPROVED** [review]
lgtm 🚀

**Pull request merged**